src/methods/wallet-send-calls.ts

```diff
@@ -21,29 +24,43 @@ import {
   validateParams,
 } from '../utils/validation';
 
+const CapabilitiesStruct = record(
+  string(),
+  type({
+    optional: optional(boolean()),
+  }),
+);
+
 const SendCallsStruct = tuple([
   object({
     version: nonempty(string()),
+    id: optional(StrictHexStruct),
     from: HexChecksumAddressStruct,
     chainId: optional(StrictHexStruct),
```

Comment:
I know this wasn't changed in this PR but is `chainId` actually optional? Doesn't appear to be in the EIP

Reply:
I understood this was our own decision as we validate against the dApp selected network, and throw if they don't match. So it's an extra convenience to rely on the dApp selected network.

Reply:
Changed to required for consistency.

Reply:
Yeah makes sense